x/oracle/abci_test.go

```diff
@@ -189,7 +195,7 @@ func (s *IntegrationTestSuite) TestEndblockerExperimentalFlag() {
 	ctx = ctx.WithBlockHeight(medianPruneBlock)
 
 	// with experimental flag off median doesn't get deleted
-	oracle.EndBlocker(ctx, app.OracleKeeper, false)
+	oracle.EndBlocker(ctx, app.OracleKeeper)
```

Comment:
We should get rid of `TestEndblockerExperimentalFlag()` now that it doesn't apply

Reply:
+1